lib/ios/ios.nit

```diff
@@ -17,3 +17,21 @@ module ios
 
 import platform
 import app
+
+import cocoa::foundation
+
+redef fun print(msg) do msg.to_s.nslog
+redef fun print_error(msg) do msg.to_s.nslog
+
+redef class Text
+	private fun nslog do to_nsstring.nslog
+end
+
+redef class NSString
+	private fun nslog in "ObjC" `{ NSLog(@"%@", self); `}
+end
+
+redef class NativeString
+	# FIXME temp workaround for #1945, bypass Unicode checks
+	redef fun char_at(pos) do return self[pos].ascii
```

Comment:
You're right, it always use the first 1 byte, so chars over more than one byte are broken. However it does the job for french characters. I looked for other workarounds, but beyond this it would probably be easier to fix the underlying problem of #1945.